src/main/kotlin/uk/gov/gdx/datashare/controller/AdminController.kt

```diff
@@ -1,4 +1,4 @@
-package uk.gov.gdx.datashare.resource
```

Comment:
I think I'd ideally have done this rename as a completely separate commit to main - it makes this a little more noisy than would be ideal

Reply:
splitting